Template profile docs fix (#845)
nf-core/tools: CHANGELOG.md

### Template

* Fixed typo in nf-core-lint CI that prevented the markdown summary from being automatically posted on PRs as a comment.

> * Fixed a mistake in template documentation regarding how profiles are loaded, and added how to identify process name for resource customisation

Reviewer: ```suggestion
* Added to template docs about how to identify process name for resource customisation
```
